### DISCUSSION TITLE
securityonion download keeps breaking

### DISCUSSION THREAD

**Discussion by darkwarriormech** [Security-Onion-Solutions/securityonion#3700] [General]
Hi all, tried downloading Securityonion many times always time out, is there any torrent that i could use to download this?
I reach maybe 3Gb of download and then it crashes cannot start from where i downloaded have to restart from scratch.

help any one?

**Comment by presianbg** [comment]
Hi,

If you are talking about the ISO file, we had the very same problem.
It turned out to be the NGFW causing this issue. 
Creating an exclusion to not scan the traffic from/to securityonion.net solved it for us.
